src/metalnx-web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -114,8 +114,6 @@
 			access="ROLE_ADMIN,ROLE_USER,ROLE_GROUPADMIN" />
 		<sec:intercept-url pattern="/preferences/**"
 			access="ROLE_ADMIN,ROLE_USER,ROLE_GROUPADMIN" />
-		<sec:intercept-url pattern="/userBookmarks/**"
-			access="ROLE_ADMIN,ROLE_USER,ROLE_GROUPADMIN" />
 		<sec:intercept-url pattern="/permissions/**"
 			access="ROLE_ADMIN,ROLE_USER,ROLE_GROUPADMIN" />
 		<sec:intercept-url pattern="/favorites/**"
```

Comment:
Favorites still in here. Is that on purpose?

Reply:
Removed.  Also removed templates.